math/odd_or_even.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file
+ * \brief Program to check if a number is an odd or even number.
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂